pubspec.lock

```diff
@@ -203,7 +203,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.16.0"
+    version: "1.15.0"
```

Comment:
can you undo these changes to pubspec.lock?

Reply:
done